test/bigtest/tests/line-edit-test.js

```diff
@@ -195,4 +195,29 @@ describe('Line edit test', () => {
       });
     });
   });
+
+  describe('Check existing Item Details Title positive case', () => {
+    const EMPTY_STRING = '';
```

Comment:
you defined constant for empty string, but below you used just `''` :)

IMO constant is not required here

Reply:
I would prefer using constant an in next commit I'll move it on common layer

Reply:
up to you

Reply:
What do you think of changing the name to more appropriate like `EMPTY_ERROR_TITLE` or something?

Reply:
this case we create constant only for using once and it might be better to leave simple ''.